fh-android-sdk/pom.xml

```diff
@@ -26,6 +26,7 @@
   <properties>
     <loopj.version>1.4.8</loopj.version>
     <aerogear.android.push.version>2.2.1</aerogear.android.push.version>
+    <aerogear.android.store.version>3.0.0-SNAPSHOT</aerogear.android.store.version>
```

Comment:
I don't feel we should be merging SNAPSHOT code into the master branch. A STABLE version should be required.
We can't read those issues. Trying <https://issues.jboss.org/browse/FH-1566> gives me a Permission Violation.